zeppelin-web/README.md

```diff
@@ -21,17 +21,18 @@ All build commands are described in [package.json](./package.json)
 
 ```sh
 # install required depepdencies and bower packages (only once)
-$ npm install
+$ npm install -g yarn
+$ yarn install
 
 # build zeppelin-web for production
-$ npm run build
+$ yarn run build
 
 # run frontend application only in dev mode (localhost:9000) 
 # you need to run zeppelin backend instance also
-$ npm run dev
+$ yarn run start 
```

Comment:
Thanks for review @AhyoungRyu 

I missed this :( since This pr was created before merging webpack PR. 
resolved in c7c240c :)